types/react-relay/lib/relay-experimental/helpers.d.ts

```diff
@@ -0,0 +1,32 @@
+/*
+A TS file to help with the construction of the official Relay (flow) types.
+ */
+
+import { FragmentReference } from 'relay-runtime';
+import { EntryPoint } from './EntryPointTypes';
+
+export type KeyType<TData = unknown> = Readonly<{
```

Comment:
I think in a future PR, we should standardize the "shape" of these files. Think relay-runtime has something similar to this. 

We can leave here for now @alloy and fix in a subsequent PR when i go through relay-runtime.

Reply:
Its got to do with that whole `$fragmentRefs`, `$refs` thing. So think once this gets merged, and I go through relay-runtime (much like this pr). Ill get a global picture of how things are meant to hang together and correct it then. For now this was needed to make this clean when authoring.